src/main/scala/org/hammerlab/guacamole/readsets/ReadSets.scala

```diff
@@ -291,7 +291,7 @@ object ReadSets extends Logging {
       result =
         result.filter(
           _.asMappedRead
-                .forall(_.alignmentQuality >= filters.minAlignmentQuality)
+                .exists(_.alignmentQuality >= filters.minAlignmentQuality)
```

Comment:
This is slightly different, this would mean that unmapped reads would be filtered.  I thought filtering unmapped reads should be separated through the range filters or passing `--mapped-only`